dali/operators/image/remap/cvcuda/matrix_adjust.cu

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/image/remap/cvcuda/matrix_adjust.h"
+
+#include <dali/core/util.h>
+#include <dali/core/geom/mat.h>
+#include <nvcv/cuda/TensorWrap.hpp>
+
+namespace dali {
+namespace warp_perspective {
+
+using MatricesWrap = nvcv::cuda::TensorWrap<float, 9 * sizeof(float), sizeof(float)>;
+
+__global__ void adjustMatricesKernel(MatricesWrap wrap, int batch_size) {
```

Comment:
It's worth mentioning that the same fixup can be applied for dst->src and src->dst mapping.

Reply:
Done